Directory.Build.props

```diff
@@ -41,8 +41,49 @@
 
     <!-- By default, nothing builds from source build. Individual projects can opt in instead -->
     <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+
+    <DefaultNetFxTargetFramework>net472</DefaultNetFxTargetFramework>
   </PropertyGroup>
 
+  <!--
```

Comment:
I had to move the TFM properties out of Versions.props in order to avoid a circular dependency:

error MSB4006: There is a circular dependency in the target dependency graph involving target "GetAllRuntimeIdentifiers".